Tests/MapboxMapsTests/MapView/Integration Tests/OfflineManagerIntegrationTests.swift

```diff
@@ -0,0 +1,158 @@
+import XCTest
+@testable import MapboxMaps
+
+internal class OfflineManagerIntegrationTestCase: MapViewIntegrationTestCase {
+
+    // MARK: Reusable test properties
+
+    /// Offline manager properties
+    private let offlineManager = OfflineManager(resourceOptions: MapInitOptions().resourceOptions)
+    private let tileRegionId = "myTileRegion"
+
+    /// Tokyo coordinates
+    private let tokyoCoord = CLLocationCoordinate2D(latitude: 35.682027, longitude: 139.769305)
+    private lazy var tokyoCoords: [CLLocationCoordinate2D] = {[
+        CLLocationCoordinate2D(latitude: tokyoCoord.latitude - 0.1, longitude: tokyoCoord.longitude - 0.1),
+        CLLocationCoordinate2D(latitude: tokyoCoord.latitude - 0.1, longitude: tokyoCoord.longitude + 0.1),
+        CLLocationCoordinate2D(latitude: tokyoCoord.latitude + 0.1, longitude: tokyoCoord.longitude + 0.1),
+        CLLocationCoordinate2D(latitude: tokyoCoord.latitude + 0.1, longitude: tokyoCoord.longitude - 0.1),
+        CLLocationCoordinate2D(latitude: tokyoCoord.latitude - 0.1, longitude: tokyoCoord.longitude - 0.1),
+    ]}()
+
+    /// Tile Region Options
+    internal var tileRegionLoadOptions: TileRegionLoadOptions?
+
+    override func setUp() {
+        super.setUp()
+
+        /// Create an offline region with tiles using the "outdoors" style
+        let stylePackOptions = StylePackLoadOptions(glyphsRasterizationMode: .ideographsRasterizedLocally,
+                                                    metadata: ["tag": "my-outdoors-style-pack"])!
+
+        _ = offlineManager.loadStylePack(for: .outdoors, loadOptions: stylePackOptions) { _ in } completion: { _ in }
```

Comment:
Do we need to wait for the style pack to load?

Reply:
nope, when I tested locally, I didn't see any timing issues